compiler/il/OMRSymbol_inlines.hpp

```diff
@@ -601,10 +601,17 @@ OMR::Symbol::isConstMethodHandle()
    return self()->isStatic() && _flags2.testAny(ConstMethodHandle);
    }
 
+void
+OMR::Symbol::setConstObjectRef()
+   {
+   TR_ASSERT(self()->isStatic(), "assert failure");
+   _flags2.set(ConstObjectRef);
+   }
+
 bool
 OMR::Symbol::isConstObjectRef()
```

Comment:
my concern here is that the set and get are not matching - isn't there a potential for confusion here? maybe the current isConstObjectRef should be come isAnyConstObjectRef? or the new flag could change name? not sure if isNonSpecificConstObjectRef or something? I just worry about the confusion of the generic quiery of is it some kind of constObjectRef and the flag where it is a const object ref not elsewhere classified.

Reply:
Renamed the flag to `NonSpecificConstObject`